### PR TITLE
Strat column edits clinton

### PR DESCRIPTION
**Pull request by clintonlunn**
placeholder for lance to view

### CODE REVIEW

**Comment by jjhi11** [comment]
@clintonlunn review please

**Comment by marshallrobinson** [comment]
@Mackenzie-Cope let's tag-team this review as this release will be the first one to be hosted on Firebase.

File: mapcontrols.js [hunk 1]
```diff
@@ -1769,7 +1769,8 @@ view.on("click", function (evt) {
     view.hitTest(evt).then((response) => {
         if (response.results.length){
             console.log('YOU CLICKED A FEATURE', response.results);
```
Comment (clintonlunn) [inline mapcontrols.js]: please remove console log @lance-weaver

File: mapcontrols.js [hunk 2]
```diff
@@ -1769,7 +1769,8 @@ view.on("click", function (evt) {
...
-            if (response.results[0].graphic.sourceLayer.id == 'ugsStratCols' || response.results[0].graphic.sourceLayer.id == 'stratCols'){
+            //if (response.results[0].graphic.sourceLayer.id == 'ugsStratCols' || response.results[0].graphic.sourceLayer.id == 'stratCols'){
+            if (response.results[0].graphic.sourceLayer.id == 'ugsStratCols'){
                 console.log('ITS A STRAT COLUMN!');
```
Comment (clintonlunn) [inline mapcontrols.js]: please remove console.log @lance-weaver

File: mapcontrols.js [hunk 3]
```diff
@@ -1769,7 +1769,8 @@ view.on("click", function (evt) {
     view.hitTest(evt).then((response) => {
         if (response.results.length){
             console.log('YOU CLICKED A FEATURE', response.results);
-            if (response.results[0].graphic.sourceLayer.id == 'ugsStratCols' || response.results[0].graphic.sourceLayer.id == 'stratCols'){
+            //if (response.results[0].graphic.sourceLayer.id == 'ugsStratCols' || response.results[0].graphic.sourceLayer.id == 'stratCols'){
```
Comment (clintonlunn) [inline mapcontrols.js:1772]: can this commented out code be removed? Are you planning to use this stratCols || ugsStratCols condition again? @lance-weaver

**Comment by Mackenzie-Cope** [comment]
Marshall and Mackenzie comments:

1. Search function for quadrangles is not working - shows an empty popup
2. Unit search: Highlighted search results do not show unit descriptions when clicked. Unit descriptions DO work outside of highlighted areas.
3. Streets and reference layer loading issue, console error.
4. Map download popup: shareable map link not copying any link
5. Map download popup: x-section icon missing
6. Map download popup: pub reference (at bottom of card) is missing scale number and says "undefined"
7. Strat columns popup on map: Change wording to "30' x 60' stratigraphic column for UGS Publication [series ID]" and "Open in a new tab"
8. Beaver quad strat column centroid is not centered in the 30x60 quad.

**Comment by marshallrobinson** [comment]
> Marshall and Mackenzie comments:
> 
> 1. Search function for quadrangles is not working - shows an empty popup
> 2. Unit search: Highlighted search results do not show unit descriptions when clicked. Unit descriptions DO work outside of highlighted areas.
> 3. Streets and reference layer loading issue, console error.
> 4. Map download popup: shareable map link not copying any link
> 5. Map download popup: x-section icon missing
> 6. Map download popup: pub reference (at bottom of card) is missing scale number and says "undefined"
> 7. Strat columns popup on map: Change wording to "30' x 60' stratigraphic column for UGS Publication [series ID]" and "Open in a new tab"
> 8. Beaver quad strat column centroid is not centered in the 30x60 quad.

Thanks for squashing most of these bugs (4 and 5 look like the only ones left). Great progress! I took a stab in item #8 (see below)
8. The Beaver quad's centroid is fixed. I took care of it in the postgres function.

**Additional comments: **

**1. Line 555 (copyright: "lance weaver",): change copyright to "Utah Geological Survey" **
**2. Lines 585-622 (// Adds national... through ...// addStratCols();): please remove**
**3. Line 675 (// if (item == "stratCols"...)): please remove**
**4. Line 560 (definition expression of series_ids): remove "m-292" (will add later)**
**5. LaSal strat column (newly added to definition expression list on line 560) doesn't work.**
**6. Please address Clinton's review comments as well if you haven't already. Thanks**

(not sure why the bolding and asterisks are formatting all weird)

**Review by marshallrobinson — APPROVED** [review]

**Pull request merged**